.env.example

```diff
@@ -57,6 +57,10 @@ PATH_OMS_SURVEY=oms-survey/docker/
 CORE_LOGIN=admin
 CORE_PASSWORD=5ecr3t
 
+SMTP_HOST=smtp.gmail.com
+SMTP_USER=admin@aegee.eu
```

Comment:
can you change it to `admin@${BASE_URL}`?

Reply:
but it's the .env file, it won't work like that, we don't have envsubst here

Reply:
you're right, then change it to `admin@DOMAIN` such that email crawlers don't get this address

Reply:
@linuxbandit one argument for force-push: if I'd do it with another commit, then it'll stay in the history so people/robots/crawlers would be able to access it